scripts/cluster/join.py

```diff
@@ -120,6 +140,7 @@ def get_connection_info(
     :param cluster_type: the type of cluster we want to join, etcd or dqlite
     :param verify_peer: flag indicating if we should verify peers certificate
     :param fingerprint: the certificate fingerprint we expect from the peer
+    :param worker: this is a worker only node
```

Comment:
In the future and in lieu of type hinting, it would be really nice to put the expected type here.  This is expected with the Sphinx style docstrings we've settled on (https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html).

Reply:
+1, I am looking forward to the PR that adds strong typing.